### ISSUE TITLE
Jack MIDI input not translated correctly

### ISSUE THREAD

**Issue by brotenet** [LMMS/lmms#5457]
Because this issue is very difficult to explain in text, I attached a .zip file with a video file that shows the issue as it happens.
[lmms-jack-midi-input-bug.mp4.zip](https://github.com/LMMS/lmms/files/4499600/lmms-jack-midi-input-bug.mp4.zip)

In all other apps (Ardour, Reaper, Carla etc.) Jack MIDI input using a simple USB MIDI keyboard (CME m-Key) works fine, but, in LMMS when I press the keys they don't get translated correctly.

When I push single notes on the MIDI keyboard, everything works fine.

When I push multiple notes (chords) on the MIDI keyboard then:
1. some notes are skipped by LMMS
2. some notes are stuck and I have to press the same note keys again to release them
3. if I press the keys of a (3 or more key) chord, only two notes are translated by LMMS

Please watch the attached video to see how it behaves in real time.

This is a really weird issue and I can't seem to find any related discussion about it either on GIT issues nor the forum.

**Comment by JohannesLorenz** [comment]
We have an open pull request #5452 fixing many JackMidi issues. I'm 99% sure it won't help with this :(

Since you seem to be an experienced JackMidi user :smiley: ... You could help us a lot if you get this PR running, enable both JackAudio and JackMidi (restart required after changing settings) and make a short test to check if anything got worse or better.

**Comment by brotenet** [comment]
> Since you seem to be an experienced JackMidi user
By all means, I'm nowhere near being an experienced user.. :sweat_smile: 

>  You could help us a lot if you get this PR running, enable both JackAudio and JackMidi (restart required after changing settings) and make a short test to check if anything got worse or better.
amm... what is a PR? and how do I get it running?? :sweat_smile:

**Comment by JohannesLorenz** [comment]
> By all means, I'm nowhere near being an experienced user.. sweat_smile

I just mean, at least, you can use it in a basic level :smiley: 

> amm... what is a PR? and how do I get it running?? sweat_smile

Pull Request. See the sections "Cloning the source code" and "compiling" of [our wiki main page](https://github.com/LMMS/lmms/wiki). Feel free to ask here or in our discord channel (linked on lmms.io) if you want to give it a try and need help.

**Comment by brotenet** [comment]
I tied building but I get an error when it tries to  get the source for x11embed.
Also, there are no errors printed in CMakeOutput.log file.

After running  `cmake ../`

```
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done

Checking submodules...
-- Checking qt5-x11embed...
-- Found Git: /usr/bin/git (found version "2.17.1") 
--   Fetching src/3rdparty/qt5-x11embed @ --depth 100
fatal: not a git repository (or any of the parent directories): .git

CMake Error at cmake/modules/CheckSubmodules.cmake:205 (MESSAGE):
  /usr/bin/git exited with status of 128
Call Stack (most recent call first):
  CMakeLists.txt:22 (INCLUDE)


-- Configuring incomplete, errors occurred!
See also "/opt/lmms/lmms-master/build/CMakeFiles/CMakeOutput.log".
```


Attachments:
[CMakeCache.txt](https://github.com/LMMS/lmms/files/4565960/CMakeCache.txt)
[CMakeOutput.log](https://github.com/LMMS/lmms/files/4565975/CMakeOutput.log)

**Comment by JohannesLorenz** [comment]
I'm sorry, I did somehow not see your message. Anyways, we have now solved that other issue, but it does not fix your issue.

So let's keep this issue open.

**Comment by brotenet** [comment]
I totally forgot about this ticket...
This issue does not exist any more because everything was solve through the magic of PipeWire! :heart: 
I literally did nothing to fix this, I'm actually running a 6.8.0-48-generic kernel and it works fine. :smiley: 
Closing..
